### PR TITLE
Tests for signing error handling

### PR DESCRIPTION
**Pull request by maackle**
Merge #756 before reviewing this

### CODE REVIEW

File: crates/holochain/src/conductor/manager/mod.rs [hunk 1]
```diff
@@ -363,7 +373,7 @@ mod test {
                     let handle = ManagedTaskAdd::ignore(handle, "respawned task");
                     TaskOutcome::NewTask(handle)
                 }
-                _ => TaskOutcome::Ignore,
+                Err(_) => unreachable!("This is not the error you're looking for"),
```
Comment (maackle) [inline crates/holochain/src/conductor/manager/mod.rs:376]:
```suggestion
                Err(_) => unreachable!("No other error is created by this test."),
```

File: crates/holochain/src/conductor/manager/mod.rs [hunk 2]
```diff
@@ -113,9 +113,12 @@ impl std::fmt::Debug for ManagedTaskAdd {
 pub enum TaskOutcome {
     /// Spawn a new managed task.
     NewTask(ManagedTaskAdd),
-    /// Ignore the exit and do nothing.
-    Ignore,
-    /// Ignore the exit and do nothing.
+    // /// Ignore the exit and do nothing.
+    // (let's not allow this for now, while we are dealing with tricky errors)
+    // Ignore,
```
Comment (maackle) [inline crates/holochain/src/conductor/manager/mod.rs:118]:
```suggestion
```

File: crates/holochain/src/conductor/manager/mod.rs [hunk 3]
```diff
@@ -167,10 +170,16 @@ async fn run(
...
                 task_manager.stream.push(new_task);
+                tracing::info!("Task added. Total tasks: {}", task_manager.stream.len());
             }
-            result = task_manager.stream.next() => match result {
+            result = task_manager.stream.next() => {
+                tracing::info!("Task completed. Total tasks: {}", task_manager.stream.len());
+                match result {
                 Some(TaskOutcome::NewTask(new_task)) => task_manager.stream.push(new_task),
-                Some(TaskOutcome::Ignore) => (),
+                // Some(TaskOutcome::Ignore) => (),
```
Comment (maackle) [inline crates/holochain/src/conductor/manager/mod.rs:179]:
```suggestion
```

**Review by neonphog — APPROVED** [review]
nice : )

**Pull request merged**